hosts/galileo/ssi/__init__.py

```diff
@@ -0,0 +1,298 @@
+################################################################################
+# hosts/galileo/ssi/__init__.py
+################################################################################
+import numpy as np
+import julian
+import cspyce
+import vicar
+import pdstable
+import pdsparser
+import oops
+
+from hosts         import pds3
+from hosts.galileo import Galileo
+
+################################################################################
+# Standard class methods
+################################################################################
+def from_file(filespec,
+              return_all_planets=False, full_fov=False, **parameters):
+    """A general, static method to return a Snapshot object based on a given
+    Galileo SSI image file.  By default, only the valid image region is
+    returned.
+
+    Inputs:
+        full_fov:           If True, the full image is returned with a mask
+                            describing the regions with no data.
+
+        return_all_planets  Include kernels for all planets not just
+                            Jupiter or Saturn.
+    """
+
+    SSI.initialize()    # Define everything the first time through; use defaults
+                        # unless initialize() is called explicitly.
+
+    # Load the PDS label
+    label = pds3.get_label(filespec)
```

Comment:
Is it possible to load an image without requiring the label, perhaps with limited functionality? We don't generally require the label for other instruments.

Reply:
We should be able to get everything from the vicar label.  I suggest moving forward with the current code right now and I will make an issue to re-do this using the VICAR label.